Add a nitpick for postgres and impersonation (#54424)
metabase/metabase: docs/permissions/impersonation.md

{% include plans-blockquote.html feature="Impersonation access" %}

> > For now, impersonation access is only available for PostgreSQL, Redshift, and Snowflake.
> > Also please consider that if you're using views in PostgresSQL, the row level security policies on views will only work on versions 15+

Reviewer: ```suggestion
> For now, impersonation access is only available for PostgreSQL, Redshift, and Snowflake.

> If you're using views in PostgresSQL, the row-level security policies on views will only work on Postgres versions 15 and higher.
```
